autonomy/chain/config.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""On-chain tools configurations."""
+
+from dataclasses import dataclass
+
+
+DEFAULT_LOCAL_RPC = "http://127.0.0.1:8545"
+DEFAULT_LOCAL_CHAIN_ID = 31337
+
+
+@dataclass
+class ContractConfig:
```

Comment:
We already have the notion of a contract config. I really don’t like how a new class is introduced for a similar concept

Reply:
If you're talking about `aea.configurations.base.ContractConfig`, it does not fit the context here since it's a config class for a component whereas this is just a standalone contract interface config class